### PR TITLE
Clean up alluxio.master.file package code

### PR DESCRIPTION
**Pull request by luzhang6**
### What changes are proposed in this pull request?
Clean up alluxio.master.file package code

Please outline the changes and how this PR fixes the issue.

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/file/FileSystemMasterJobServiceHandler.java [hunk 1]
```diff
@@ -54,9 +54,9 @@ public void getFileInfo(GetFileInfoPRequest request,
     GetFileInfoPOptions options = request.getOptions();
 
     RpcUtils.call(LOG,
-        (RpcUtils.RpcCallableThrowsIOException<GetFileInfoPResponse>) () -> GetFileInfoPResponse
-            .newBuilder().setFileInfo(GrpcUtils.toProto(mFileSystemMaster.getFileInfo(fileId)))
-            .build(),
+            () -> GetFileInfoPResponse
```
Comment (yuzhu) [inline core/server/master/src/main/java/alluxio/master/file/FileSystemMasterJobServiceHandler.java:57]: spacing issues?

File: core/server/master/src/main/java/alluxio/master/file/NoopBlockDeletionContext.java [hunk 2]
```diff
@@ -19,11 +19,9 @@ public final class NoopBlockDeletionContext implements BlockDeletionContext {
 
   @Override
   public void registerBlockForDeletion(long blockId) {
-    return;
```
Comment (yuzhu) [inline core/server/master/src/main/java/alluxio/master/file/NoopBlockDeletionContext.java:22]: typically we add a comment explicitly saying it is a noop, not that we forgot to implement.

**Review by yuzhu — CHANGES_REQUESTED** [review]

File: core/server/master/src/main/java/alluxio/master/file/NoopBlockDeletionContext.java (hunk 2 above)
Comment (luzhang6) [reply core/server/master/src/main/java/alluxio/master/file/NoopBlockDeletionContext.java:22]: will add comment

File: core/server/master/src/main/java/alluxio/master/file/FileSystemMasterJobServiceHandler.java (hunk 1 above)
Comment (luzhang6) [reply core/server/master/src/main/java/alluxio/master/file/FileSystemMasterJobServiceHandler.java:57]: It's just the casting '() -> {...}' to 'RpcUtils.RpcCallableThrowsIOException<GetFileInfoPResponse>' is redundant.

**Comment by luzhang6** [comment]
@yuzhu can you review this again and merge if no issues? thanks!

**Review by yuzhu — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**